src/components/CalculatorSettings.js

```diff
@@ -0,0 +1,188 @@
+                    { value: 1, label: '1' },
+                    { value: 2, label: '2' },
+                    { value: 3, label: '3' },
+                    { value: 4, label: '4' },
+                    { value: 5, label: '5' },
+                    { value: 6, label: '6' },
+                    { value: 7, label: '7' },
+                ]}
+                selection={calculatorTier}
+                setSelection={val => dispatch(updateCalculatorsTier(val))}
+            />
+            <h3 className='heading-accent-md mt-4'>Skill</h3>
+            <Select
+                className='w-full'
+                onSelect={e => dispatch(updateCalculatorsSkill({ skill: e.value }))}
+                options={calculatorSkills}
+                value={selectedSkill}
+            />
+            <h3 className='heading-accent-md mt-4'>Experience</h3>
+            <div className='grid grid-cols-2 gap-4'>
+                <div>
+                    <h4>Start</h4>
+                    <Select
+                        onSelect={selection => updateMode(selection, 'start')}
+                        options={[
+                            { label: 'Experience', value: 'xp' },
+                            { label: 'Level', value: 'level' },
+                        ]}
+                        value={expValues.start.mode}
+                    />
+                    <input
+                        className='input-primary form-input mt-2 w-full'
+                        onChange={e => updateXpValue(e, 'start')}
+                        maxLength={expValues.start.mode === 'level' ? 2 : 9}
+                        type='text'
+                        value={expValues.start[expValues.start.mode]}
+                    />
+                </div>
+
+                <div>
+                    <h3>End</h3>
+                    <Select
+                        onSelect={selection => updateMode(selection, 'target')}
+                        options={[
+                            { label: 'Level', value: 'level' },
+                            { label: 'Experience', value: 'xp' },
+                        ]}
+                        value={expValues.target.mode}
+                    />
+                    <input
+                        className='input-primary form-input mt-2 w-full'
+                        onChange={e => updateXpValue(e, 'target')}
+                        maxLength={expValues.target.mode === 'level' ? 2 : 9}
+                        type='text'
+                        value={expValues.target[expValues.target.mode]}
+                    />
+                </div>
+            </div>
+            <p className={isValidValues ? 'text-accent' : 'text-error'}>
+                {isValidValues
+                    ? `XP required: ${numberWithCommas(xpRequired)}`
+                    : 'Start experience must be lower than end'}
+            </p>
+
+            {/* TODO
+                - Fragment/Set effect overrides
+            */}
```

Comment:
Could put this as a "coming soon" note on the UI so people don't immediately come to the discord suggesting to add this 😅